fern/docs/pages/welcome/quickstart.mdx

```diff
@@ -89,10 +85,10 @@ groups:
   local:
     generators:
       - name: fernapi/fern-typscript-node-sdk
-          version: 0.8.8
+          version: 0.x.x
           output:
             location: local-file-system
-            path: ../generated/sdks/node
+            path: ../generated/sdks/typescript
```

Comment:
should we fix the indentation here? `name / version / output` should be on the same level

Reply:
great eye!